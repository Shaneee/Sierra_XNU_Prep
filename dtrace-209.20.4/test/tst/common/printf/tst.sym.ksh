#!/bin/sh



# ASSERTION:
#  Test printf() with a simple string argument.
#
#  SECTION: Output Formatting/printf()
#
#

script()
{
	dtrace -qs /dev/stdin <<EOF
	BEGIN
	{
		printf("symbol = %a", &\`real_ncpus);
		exit(0);
	}
EOF
}

script | tee /dev/fd/2 | grep -E  'kernel.+real_ncpus' > /dev/null

exit $?
