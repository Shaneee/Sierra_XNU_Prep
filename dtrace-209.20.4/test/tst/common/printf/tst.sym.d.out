symbol = mach_kernel`real_ncpus
